src/04-calendar/events.apib

```diff
@@ -16,6 +16,7 @@ Get a list of calendar events.
             + activity_type_id: `edd94120-63e2-4b10-80ed-fdfcfaa0a515` (string, optional)
             + ends_after: `2017-01-01T00:00:00+00:00` (string, optional) - Start of the period for which to return events
             + starts_before: `2018-01-01T00:00:00+00:00` (string, optional) - End of the period for which to return events
+            + contact_id: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5` (string, optional)
```

Comment:
Won't we need the possibility of requestion events for multiple contacts? for e.g. comparing a set of contacts to see if they are free at the same date?

Reply:
You can easily do this with multiple requests for now?